### PR TITLE
Global-CS oversampling

### PR DESCRIPTION
**Pull request by plutasnyy**

### CODE REVIEW

File: multi_imbalance/resampling/tests/test_globalcs.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from collections import Counter, defaultdict
+
+import numpy as np
+import pytest
+
+from multi_imbalance.resampling.GlobalCS import GlobalCS
+from multi_imbalance.resampling.SOUP import SOUP
```
Comment (plutasnyy) [inline multi_imbalance/resampling/tests/test_globalcs.py]: unused import, to remove

Comment (plutasnyy) [reply multi_imbalance/resampling/tests/test_globalcs.py]: done

File: multi_imbalance/resampling/GlobalCS.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from sklearn.neighbors import NearestNeighbors
+
+
+class GlobalCS(object):
+    """
+    Global CS is an algorithm that equalizes number of samples in each class. It duplicates all samples equally
+    for each class to achieve majority class size
+    """
+
+    def __init__(self) -> None:
```
Comment (damian-horna) [inline multi_imbalance/resampling/GlobalCS.py]: Do we need this type hint? __init__ by definition should not return anything, so I thinks it's useless here

File: multi_imbalance/resampling/GlobalCS.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        Parameters
+        ----------
+        X two dimensional numpy array (number of samples x number of features) with float numbers
+        y one dimensional numpy array with labels for rows in X
+
+        Returns
+        -------
+        Resampled X (max class quantity * number of unique classes), y (number of rows in X) as numpy array
+        """
+        assert len(X.shape) == 2, 'X should have 2 dimension'
```
Comment (damian-horna) [inline multi_imbalance/resampling/GlobalCS.py]: at least 2 samples?

**Pull request merged**